docs/how-to-guides/add-conditional-control-to-a-flow.md

```diff
@@ -59,7 +59,7 @@ activate:
   ![bypassed_nodes](../media/how-to-guides/conditional-flow-with-activate/bypassed_nodes.png)
 
 
-    (1) If a node has activate config and the value of `activate.when` is equals to `activate.is`, it will be bypassed.
+    (1) If a node has activate config and the value of `activate.when` is not equals to `activate.is`, it will be bypassed.
```

Comment:
We should also say that it will always be executed when the condition is met.

Reply:
I think the logic here is that, if activate config is set, we will always use this config.